### DISCUSSION TITLE
MET/11.0.0; conda env on Cheyenne

### DISCUSSION THREAD

**Discussion by junmeiban** [dtcenter/METplus#2030] [Existing Builds] [answered]
Dear MET_help,

I am trying to run MET using conda env on Cheyenne. I run the following command:

source /glade/u/apps/ch/modulefiles/default/localinit/localinit.sh
module purge
module use /glade/p/ral/jntp/MET/MET_releases/modulefiles
module load met/11.0.0
module load conda/latest
conda activate npl

Then, I got the following error :
whence: -t: unknown option

Any idea of the error? Do you know how to fix this? or should I send this to CISL help?
Thank you so much for your help,
Junmei

**Comment by jprestop** [comment, marked as answer]
HI @junmeiban.

I am unable to replicate your error on Cheyenne.  Have you tried starting fresh with a new login?  Here is what I get when I run those same commands on Cheyenne and then test to ensure the MET tools are loaded and provide a usage statement on a call to "--help":
```
(base) jpresto@cheyenne5:~$ source /glade/u/apps/ch/modulefiles/default/localinit/localinit.sh
(base) jpresto@cheyenne5:~$ module purge
(base) jpresto@cheyenne5:~$ module use /glade/p/ral/jntp/MET/MET_releases/modulefiles
(base) jpresto@cheyenne5:~$ module load met/11.0.0
(base) jpresto@cheyenne5:~$ module load conda/latest
(base) jpresto@cheyenne5:~$ conda activate npl
(npl) jpresto@cheyenne5:~$ which point_stat
/glade/p/ral/jntp/MET/MET_releases/11.0.0/bin/point_stat
(npl) jpresto@cheyenne5:~$ point_stat --help
DEBUG 1: Start point_stat by jpresto(7414) at 2023-01-31 19:04:38Z  cmd: point_stat --help 

*** Model Evaluation Tools (METV11.0.0) ***

Usage: point_stat
	fcst_file
	obs_file
	config_file
	[-point_obs file]
	[-obs_valid_beg time]
	[-obs_valid_end time]
	[-outdir path]
	[-log file]
	[-v level]

	where	"fcst_file" is a gridded forecast file containing the field(s) to be verified (required).
		"obs_file" is an observation file in NetCDF format containing the verifying point observations (required).
		"config_file" is a PointStatConfig file containing the desired configuration settings (required).
		"-point_obs file" specifies additional NetCDF point observation files to be used (optional).
		"-obs_valid_beg time" in YYYYMMDD[_HH[MMSS]] sets the beginning of the matching time window (optional).
		"-obs_valid_end time" in YYYYMMDD[_HH[MMSS]] sets the end of the matching time window (optional).
		"-outdir path" overrides the default output directory (.) (optional).
		"-log file" outputs log messages to the specified file (optional).
		"-v level" overrides the default level of logging (2) (optional).

(npl) jpresto@cheyenne5:~$ 
```

Please try again, starting fresh with a new login and let us know how it goes.

**Reply by junmeiban** [reply]
Thanks @jprestop ! It works. This question can be closed.

**Reply by jprestop** [reply]
Excellent, @junmeiban!  Thank you for testing and for letting us know.  I will go ahead and mark this as Answered and will lock the Discussion to prevent future posts.  Please feel free to start a new Discussion should any new issues arise.
